src/cmsAdapter.js

```diff
@@ -1,11 +1,23 @@
 const getTenScreeningsAdapter = {
   async loadAllMovieScreenings() {
     const response = await fetch(
-      'https://plankton-app-xhkom.ondigitalocean.app/api/screenings?populate=movie'
+      "https://plankton-app-xhkom.ondigitalocean.app/api/screenings?populate=movie"
     );
     const payload = await response.json();
     return payload.data;
   },
 };
 
-export default getTenScreeningsAdapter;
+export { getTenScreeningsAdapter };
+
+const cmsAdapter = {
```

Comment:
This looks like a good CMS adapter, but I would put all of the adapter methods on a single object/class, instead of having multiple adapters that all represent the CMS.